app/controllers/bridges_controller.rb

```diff
@@ -1,4 +1,49 @@
 # frozen_string_literal: true
 
 class BridgesController < ApplicationController
+  before_action :authorize_request
+  before_action :set_bridge, only: %i[show update destroy]
+
+  def index
+    render_message message: @current_user.bridges.all
+  end
+
+  def show
+    render_message message: @bridge.to_json(include: %i[headers environment_variables events])
+  end
+
+  def create
+    @bridge = Bridge.new(bridge_params)
+    @bridge.user = @current_user
+
+    if @bridge.save
+      render_message status: :created
+    else
+      render_message message: @bridge.errors, status: :internal_server_error
+    end
+  end
+
+  def update
+    if @bridge.update bridge_params
+      render_message
+    else
+      render_message message: @bridge.errors, status: :internal_server_error
+    end
+  end
+
+  def destroy
+    @bridge.destroy
+    render_message
+  end
+
+  protected
+
+  def bridge_params
+    params.require(:bridge).permit(:name, :method, :retries, :delay, :outbound_url, :payload)
```

Comment:
You need to accept nested params for env vars and headers. Remember, i will be sending bridge, editor code, headers & env vars into this create method. Look up `accepts_nested_params_for`.

Reply:
Oh and you can't just pass params into bridge and save. You have to build the data column. 

I send a json payload with "testPayload" & "payload", you need to save them into 1 column like 

```ruby
{
  testPayload: 
  payload:
}
```